### DISCUSSION TITLE
Selecting aperture for RTM

### DISCUSSION THREAD

**Discussion by kerim371** [slimgroup/JUDI.jl#150] [General]
Hi,

I get RTM like in the picture below. 
I suspect I need to set an aperture for migration.
Is it possible? Or the problem isn't connected with aperture?
```julia
# Setup operators
Pr = judiProjection(d_obs.geometry)
F = judiModeling(model0; options=jopt)
Ps = judiProjection(q.geometry)
J = judiJacobian(Pr*F*adjoint(Ps), q)

# RTM
rtm = adjoint(J)*d_obs
```
![image](https://user-images.githubusercontent.com/43808863/198129767-904bb178-0145-4b3a-94e9-a385c3786ee3.png)

**Comment by mloubout** [comment]
What are your options `jopt `? Looks like usual FWI artifacts that can be removed with isic or a Laplacian filter

**Comment by kerim371** [comment]
RTM options:
```julia
# JUDI options
jopt = JUDI.Options(
    save_data_to_disk=true, 
    file_path=shots_out_dir, 
    file_name=shots_out_name,
    limit_m = true,
    buffer_size = 100f0,
    optimal_checkpointing=false)
```

FWI options:
```julia
jopt = JUDI.Options(
    save_data_to_disk=true, 
    file_path=shots_out_dir, 
    file_name=shots_out_name,
    limit_m = true,
    buffer_size = 100f0,
    optimal_checkpointing=false)
```

Buffersize is such small due to RAM limitations. 

Model computed with FWI
![image](https://user-images.githubusercontent.com/43808863/198131598-e74cad59-2caf-4c7b-b184-29b3f7cd150a.png)

**Reply by mloubout** [reply]
You may wanna try `isic=true` for RTM

**Comment by kerim371** [comment]
Thank you!
Now it looks much better.

What is possible the reason of random noise (especially in the upper part)?
![image](https://user-images.githubusercontent.com/43808863/198153240-23099727-753c-42e8-81d4-a3a9f9b59d47.png)

**Reply by mloubout** [reply]
Sweet
